care_o_bot_simulation/package.xml

```diff
@@ -10,7 +10,7 @@
 
   <buildtool_depend>catkin</buildtool_depend>
 
-  <run_depend>cob_simulation</run_depend>
+  <exec_depend>cob_simulation</exec_depend>
```

Comment:
As for `care_o_bot_robot` this should just contain
```
<exec_depend>cob_bringup_sim</exec_depend>
```

Additionally we could add
```
<exec_depend>cob_navigation</exec_depend>
<exec_depend>cob_manipulation</exec_depend>
```